src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ( props) => {
```

Comment:
Your spacing is inconsistent throughout this file, consider setting up your linter to catch these style issues so you can fix them before opening a PR.

```suggestion
const ChatLog = (props) => {
```